<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>bulkDetailsDialog</class>
    <widget class="QDialog" m_name="bulkDetailsDialog">
        <property m_name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>494</width>
                <height>220</height>
            </rect>
        </property>
        <property m_name="font">
            <font>
                <pointsize>10</pointsize>
            </font>
        </property>
        <property m_name="windowTitle">
            <string>Bulk Details</string>
        </property>
        <property m_name="windowIcon">
            <iconset resource="../resources/images.qrc">
                <normaloff>:/icons/books_icon.png</normaloff>:/icons/books_icon.png
            </iconset>
        </property>
        <layout class="QVBoxLayout" m_name="verticalLayout">
            <item>
                <widget class="QLabel" m_name="labelPrompt">
                    <property m_name="text">
                        <string>If you wish all ebooks in the selected folder to share the below data, check the boxes
                            and input your specification:
                        </string>
                    </property>
                    <property m_name="wordWrap">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QGridLayout" m_name="gridLayout">
                    <item row="1" column="0">
                        <widget class="QCheckBox" m_name="checkBoxGenre">
                            <property m_name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QLabel" m_name="labelTags">
                            <property m_name="text">
                                <string>Tags:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="0">
                        <widget class="QCheckBox" m_name="checkBoxAuthor">
                            <property m_name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QCheckBox" m_name="checkBoxTags">
                            <property m_name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLabel" m_name="labelAuthor">
                            <property m_name="text">
                                <string>Author:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLabel" m_name="labelGenre">
                            <property m_name="text">
                                <string>Genre:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="2">
                        <widget class="QLineEdit" m_name="textAuthor">
                            <property m_name="enabled">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="2">
                        <widget class="QLineEdit" m_name="textGenre">
                            <property m_name="enabled">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="2">
                        <widget class="QLineEdit" m_name="textTags">
                            <property m_name="enabled">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" m_name="horLayBottomButtons">
                    <item>
                        <spacer m_name="horizontalSpacer">
                            <property m_name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property m_name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" m_name="buttonApply">
                            <property m_name="minimumSize">
                                <size>
                                    <width>80</width>
                                    <height>25</height>
                                </size>
                            </property>
                            <property m_name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property m_name="text">
                                <string>Apply</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" m_name="buttonCancel">
                            <property m_name="minimumSize">
                                <size>
                                    <width>80</width>
                                    <height>25</height>
                                </size>
                            </property>
                            <property m_name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property m_name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="../resources/images.qrc"/>
    </resources>
    <connections/>
</ui>
